localstack-core/localstack/aws/handlers/validation.py

```diff
@@ -45,6 +73,7 @@ class OpenAPIRequestValidator(OpenAPIValidator):
     """
 
     def __call__(self, chain: HandlerChain, context: RequestContext, response: Response):
+        self._record_usage(context)
```

Comment:
As this executed as the first step of the handler, this will be called for every request hitting LocalStack, even if not an internal route. I believe this should be put after the following condition:
`if path.startswith(f"{INTERNAL_RESOURCE_PATH}/") or path.startswith("/_aws/"):`

Reply:
Absolutely!